### DISCUSSION TITLE
Allow select input for data-ajax-cart-quantity-input

### DISCUSSION THREAD

**Discussion by leo-cheron** [EvgeniyMukhamedjanov/liquid-ajax-cart#98] [Ideas]
Currently [only "text" or "number" inputs are allowed](https://github.com/EvgeniyMukhamedjanov/liquid-ajax-cart/blob/main/_src/controls/quantity-input.ts#L35). "select-one" input type is better suited in many cases, and it would make sense to also allow it.

**Comment by EvgeniyMukhamedjanov** [comment]
Hi @leo-cheron ,

Thank you for noticing.
I've added HTMLSelecteElement check to the same line but forgot to add type for it.
I saw you fixed it on your branch. Did you test if that works for you?
Can you make a PR, I'll test it as well.

**UPD** Ignore me, I've just added this feature to the upcoming release :-)

**Reply by leo-cheron** [reply]
Perfect, thanks!
